banking/src/main/java/rocks/kata/CurrentDateProvider.java

```diff
@@ -0,0 +1,12 @@
+package rocks.kata;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class CurrentDateProvider {
```

Comment:
jak masz już taką ładną klasę z metodą `getCurrentDate`, to:
1. nazwa mogłaby być bardziej ogólna (`DateProvider`) - dzięki temu kolejne metody można dodawać w przyszłości bez problemu
2. wydaje się, że format daty powinien być podawany jako argument dla poprawy elastyczności